.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would say the same as having a separated `webpack.config.js` and `.eslintrc`: potentially better editor support. My reasoning: an editor MIGHT now about these files but it will NEVER know about objects in random files.